recipes/mingw/all/conanfile.py

```diff
@@ -0,0 +1,73 @@
+import os
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class MingwConan(ConanFile):
+    name = "mingw"
+    description = "MinGW is a contraction of Minimalist GNU for Windows"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.mingw.org"
```

Comment:
This site does not load for me

Reply:
I'll move to http://mingw-w64.org/doku.php